release-team/README.md

```diff
@@ -185,3 +185,19 @@ is undefined.
 ### During "Security" releases
 For all unplanned or embargoed releases
 - Facilitate security releases following the under the [Security Release Process](security-release-process-documentation/security-release-process.md)
+
+---
+
+## Milestone Maintainers
+
+Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
```

Comment:
Not trying to be the well actually guy, but the team is actually called [`kubernetes-milestone-maintainers`](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers)

We gate `/status` and `/milestone` behind this team.  [Currently for _all_ repos in the kubernetes org](https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L160-L167).  We can configure this on a per-repo basis if there were a desire to do so (ie: I'm thinking about opening up test-infra to a different team of people)

AFAIK the main place the release team cares about these commands is `kubernetes/kubernetes`, I have historically not seen a ton of involvement in the `kubernetes/features` repo.  I would use devstats to find out, but apparently it's not tracking either of these bot commands.

Reply:
The main motivation I saw behind this team was to ensure that not any-old-random dev could shove their code into a milestone, once it came time for code freeze.  I personally don't find the `status/` labels helpful at all.